OpenSim/Common/Exception.cpp

```diff
@@ -67,6 +69,23 @@ exception()
 //#endif
 }
 
+Exception::Exception(const std::string& file,
+                     size_t line,
+                     const std::string& func) {
+    addMessage(file + ":" + std::to_string(line) + _msgSeparator + 
```

Comment:
I am not sure I understand.
How is "what is appropriate?" determined here when it has never been done so far ?
How do we know that having file/line/func at the end is appropriate ?